src/pages/deployments/openshift.mdx

```diff
@@ -1,5 +1,17 @@
 ---
 title: Deploy on Openshift
+description: Instructions to deploy on Openshift
 ---
-Prerequisites: Openshift 4.3, Cloud Pak for Applications, Kabanero 
+
+## Pre-requisites: 
+
+Openshift 4.3, Cloud Pak for Applications, Kabanero 
+
+### Create an organization on Github 
+
+### Fork the repository with the script 
+
+### Create a Webhook for the script 
```

Comment:
We don't need this section at the moment

Additionally add one more section to include the instructions for deploying the databases